src/meta/src/stream/stream_manager.rs

```diff
@@ -864,7 +867,11 @@ mod tests {
         async fn drop_materialized_views(&self, table_ids: Vec<TableId>) -> MetaResult<()> {
             for table_id in &table_ids {
                 self.catalog_manager
-                    .drop_table(table_id.table_id, vec![], self.fragment_manager.clone())
+                    .drop_relation(
+                        RelationIdEnum::Table(table_id.table_id),
+                        self.fragment_manager.clone(),
+                        DropMode::Restrict,
```

Comment:
Should this be inheriting `DropMode` setting from the caller?

Reply:
It is a test method of `MockService` and so far so good.